packages/router/src/history.tsx

```diff
@@ -1,20 +1,22 @@
 const createHistory = () => {
-  const listeners = {}
+  type Listener = (ev?: PopStateEvent) => any
+
+  const listeners: Record<string, Listener> = {}
 
   return {
-    listen: (listener) => {
+    listen: (listener: Listener) => {
       const listenerId = 'RW_HISTORY_LISTENER_ID_' + Date.now()
       listeners[listenerId] = listener
       window.addEventListener('popstate', listener)
       return listenerId
     },
-    navigate: (to) => {
-      window.history.pushState({}, null, to)
+    navigate: (to: string) => {
+      window.history.pushState({}, '', to)
```

Comment:
This is the title right? I'm guessing this has to be an empty string and can't be `undefined`?

Reply:
![image](https://user-images.githubusercontent.com/30793/106502849-2eb6d100-64c5-11eb-8876-f8f0f92a75a3.png)

It's non-optional, so have to be an empty string